sufia-models/app/models/concerns/sufia/generic_file/querying.rb

```diff
@@ -0,0 +1,39 @@
+module Sufia
+  module GenericFile
+    module Querying
+      extend ActiveSupport::Concern
+
+      module ClassMethods
+        # query to find generic files created during the time range
+        # @param [DateTime] start_datetime starting date time for range query
+        # @param [DateTime] end_datetime ending date time for range query
+        def find_by_date_created(start_datetime, end_datetime=nil)
+          return [] if start_datetime.blank?  # no date just return nothing
+          start_date_str =  start_datetime.utc.strftime("%Y-%m-%dT%H:%M:%SZ")
+          end_date_str = if end_datetime.blank?
+                           "*"
+                         else
+                           end_datetime.utc.strftime("%Y-%m-%dT%H:%M:%SZ")
+                         end
+          where "system_create_dtsi:[#{start_date_str} TO #{end_date_str}]"
```

Comment:
Ha! Here's a good guide for when to use and not use parens: https://github.com/bbatsov/ruby-style-guide